deploy/runtime/webapis.conf

```diff
@@ -2,6 +2,8 @@
 ' Modifying this file may cause SAM's download features to stop working.
 
 android_build=https://sam.nrel.gov/sites/default/files/content/mobile/android/readme.html
+cambium_meta=https://cambium.nrel.gov/api/project/detail/?uuid=579698fe-5a38-4d7c-8611-d0c5969b2e54
```

Comment:
Can we place the uuid in private.h in the SAM-private repo? Was this uuid assigned by Cambium and does it have a monthly bandwidth limit?

Reply:
The uuid just differentiates which Cambium Data Viewer project that data is pulled from.  I don't think it's necessarily the same as an API key, and I don't know if they have any monthly bandwidth limits on their projects.  I can ask Robert about that.  I can place it in the SAM-private repo if you think that would be better or remove confusion later though.

Reply:
I confirmed that there is no monthly bandwidth limit currently on the Cambium projects.